src/simulations/ensemblerun.jl

```diff
@@ -19,6 +19,10 @@ docs online).
 All other keywords are propagated to [`run!`](@ref) as-is.
 
 Example usage in [Schelling's segregation model](@ref).
+
+If you want to scan parameters and at the same time run multiple simulations at each
```

Comment:
Was this added in the parallel paramscan PR already?

Reply:
This was always possible de-fact actually, as `seed` is nothing more as just another model parameter that can be "scanned" over.

Reply:
(of course the user has to actually add this seed to the model by `ABM(...; rng = MersenneTwister(seed))` but I hope they are intelligent enough to figure this out :D )